src/.gitignore

```diff
@@ -0,0 +1,398 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
These changes are both unrelated to the commit message and pointless - don't use generated ignore files that don't apply to the project in question!

Reply:
I have moved the needed ignore path to the root gitignore file